<launch>
	<arg name="publish_rate" default="100"/>
	<arg name="x_point" default="50"/>
	<arg name="y_point" default="50"/>
	<arg name="block_num" default="49"/>

	<node name="pharos_bkmap" pkg="pharos_bkmap" type="pharos_bkmap" output="screen">
		<!--<param name="publish_rate" value="$(arg publish_rate)"/>
		<remap from="test_pub" to="test_pub" />
		<remap from="test_sub" to="test_sub" />-->
		<param name="x_point" value="$(arg x_point)"/>
		<param name="y_point" value="$(arg y_point)"/>
		<param name="block_num" value="$(arg block_num)"/>
	</node>

	<!-- <include file="$(find pharos_pkg)/launch/pharos_launch.launch">
		<arg name="param1" value="$(arg arg1)" />
	</include> -->

</launch>